src/utils/styleUtils.ts

```diff
@@ -11,16 +11,43 @@ export function getRowStyle(rowIdx: number, height?: number): CSSProperties {
       '--rdg-row-height': `${height}px`
     } as unknown as CSSProperties;
   }
+
   return { '--rdg-grid-row-start': rowIdx } as unknown as CSSProperties;
 }
 
+export function getHeaderCellStyle<R, SR>(
```

Comment:
Worth updating this function to handle `colSpan` also?

Reply:
`getCellStyle` handles it already, so it'd duplicate the work for normal header cells 🤔